packages/shared/src/components/filters/TagsFilter.tsx

```diff
@@ -23,6 +24,23 @@ import { FilterMenuProps } from './common';
 import MenuIcon from '../../../icons/menu.svg';
 import AuthContext from '../../contexts/AuthContext';
 import { useMyFeed } from '../../hooks/useMyFeed';
+import { Features, getFeatureValue } from '../../lib/featureManagement';
+import FeaturesContext from '../../contexts/FeaturesContext';
+
+const containerClass = {
+  v3: 'flex-col-reverse',
+  v5: 'flex-col-reverse',
+};
+
+const searchFieldClass = {
+  v3: 'mt-6 mb-2',
+  v5: 'mb-2',
```

Comment:
As you mentioned, you wanted to override the bottom margin of `v5`. Why not put it here instead?

Reference: https://github.com/dailydotdev/apps/pull/928/files#r833365205
```suggestion
  v5: 'mb-6',
```

Reply:
It seems to me that margin 2 is the correct one because there is also a paragraph with its padding of 6. For version 5 margin should be 8.

Reply:
Alright, thought you really wanted the 6 since you mentioned it in your reply to Chris. This should be okay now as you have verified.